### PR TITLE
[IMP] sales: updated import products doc for 16

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3496197&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### FWP up to `master`

### CODE REVIEW

File: content/applications/sales/sales/products_prices/products/import.rst [hunk 1]
```diff
@@ -1,43 +1,264 @@
...
+
+When all desired relation field information has been entered, save the spreadsheet, and import it
+to the database, per the process mentioned above (:menuselection:`Sales app --> Products -->
+Products --> ⚙️ (gear) icon --> Import records --> Upload File`).
+
+Once the spreadsheet with the newly-configured relation field information has been uploaded, click
+:guilabel:`Import`, and Odoo returns to the :guilabel:`Products` page.
+
+When the newly-changed/modified products (complete with the new relation field information) has been
+imported and uploaded, that new information can be found on that product's product page.
```
Comment (jero-odoo) [inline content/applications/sales/sales/products_prices/products/import.rst:166]:
```suggestion
imported and uploaded, that new information can be found on the :guilabel:`Products` page.
```
I understand you are trying to refer to an individual product record (page) here, so I do not know if this is the best wording, but I don't think "product's product page" works. Maybe just "Product record"?

File: content/applications/sales/sales/products_prices/products/import.rst [hunk 2]
```diff
@@ -1,43 +1,264 @@
...
+
+Attributes and values
+---------------------
+
+Odoo also allows users to import product attributes and values that can be used for products that
+already exist in the database and/or with imported products.
+
+To import attributes and values, a separate Excel or CSV file dedicated to attributes and values
```
Comment (jero-odoo) [inline content/applications/sales/sales/products_prices/products/import.rst:174]:
```suggestion
To import attributes and values, a separate spreadsheet or CSV file dedicated to attributes and values
```

File: content/applications/sales/sales/products_prices/products/import.rst [hunk 3]
```diff
@@ -1,43 +1,264 @@
...
+time to import and upload them into Odoo. To do that, navigate to :menuselection:`Sales app -->
+Products --> Products --> ⚙️ (gear) icon --> Import records --> Upload File`.
+
+Once the spreadsheet with the newly-configured products, and product variants, has been uploaded,
+click :guilabel:`Import`, and Odoo returns to the :guilabel:`Products` page. That's where the
+newly-added products can be found.
+
+To view and modify the product variants on any products, select the desired product from the
+:guilabel:`Products` page, and click the :guilabel:`Attributes & Variants` tab. That's where the
+product attributes and variants can be found and modified, if necessary.
```
Comment (jero-odoo) [inline content/applications/sales/sales/products_prices/products/import.rst:261]:
```suggestion
To view and modify the attributes and variants on any products, select the desired product from the
:guilabel:`Products` page, and click the :guilabel:`Attributes & Variants` tab.
```

**Review by jero-odoo — CHANGES_REQUESTED** [review]
@ksc-odoo Great work! Just a few comments, let me know if you have any questions! 👍

**Comment by ksc-odoo** [comment]
Thanks for all your helpful comments/feedback, @jero-odoo -- made all the suggested edits, and once you give this your approval, I'll then tag the materials team, so they can conduct a 2nd round of peer review for this PR... 👍

**Review by jero-odoo — APPROVED** [review]

**Comment by jero-odoo** [comment]
Hey @ksc-odoo this looks good to me 👍

**Comment by ksc-odoo** [comment]
Thanks, @jero-odoo ! Alrighty, Materials Team -- This one is ready for a 2nd Round of Peer Review (@hojo-odoo @Felicious @brse-odoo ) 👍

**Review by brse-odoo — COMMENTED** [review]
@ksc-odoo Just finished reviewing this PR. Great job! Once again, I didn't have many suggestions outside of grammatical suggestions and the potential for additional information, for you to accept/reject as you see fit.

Feel free to ping me with any questions, thanks!

**Review by brse-odoo — APPROVED** [review]

**Comment by ksc-odoo** [comment]
Thanks for the helpful feedback @brse-odoo - made all the suggested edits! So now this one is ready for Final Review @odoo/us-doc-review 👍

**Comment by samueljlieber** [comment]
Hi @StraubCreative this PR is good to go 👍

### FWP up to master

**Review by StraubCreative — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**